.github/workflows/provenance.yaml

```diff
@@ -0,0 +1,21 @@
+name: Generate Provenance
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  generate_provenance:
+    permissions:
+      actions: read
+      id-token: write
+      contents: write
+    uses: slsa-framework/slsa-github-generator/.github/workflows/builder_docker-based_slsa3.yml@main
+    with:
+      builder-image: "europe-west2-docker.pkg.dev/oak-ci/hello-transparent-release/hello-transparent-release"
+      builder-digest: "sha256:eb0297df0a4df8621837369006421dd972cc3e68e6da94625539f669d49f1525"
```

Comment:
Nice, I think this connects with the description in https://github.com/project-oak/hello-transparent-release#build-a-builder-docker-image .

Do you plan to update the README, too?

Reply:
Yes. Once this is working I'll update the README too :)